fetch.bs

```diff
@@ -6095,7 +6094,13 @@ that RFC's normative processing requirements to be compatible with deployed cont
 
    <li><p>If <var>body</var> is failure, then return failure.
 
-   <li><p>Remove the last 7 code points from <var>mimeType</var>.
+   <li><p>Remove trailing U+0020 SPACE <a>code points</a> from <var>mimeType</var>, if any.
```

Comment:
Do you think it'd be worth pointing out that just not implementing these steps would have no observable effect, since the MIME parser will ignore it? Or if it can make a difference (didn't check deeply) pointing that out, to avoid someone making that optimization.

Reply:
Yeah that sounds like a good idea. I mainly clean up here in case we want to add some kind of error reporting to the MIME type parser later.